repository-data/webfiles/src/main/resources/site/scss/components/_update-box.scss

```diff
@@ -0,0 +1,131 @@
+/* Update box */
+$txt-left-padding-desk: 125px;
+$txt-left-padding-mob: 50px;
+$box-padding: 20px;
+
+.update-box {
+    position: relative;
+    padding: $box-padding;
+    border-left: 7px solid;
+
+    &-group {
+        margin-bottom: $default-spacing-unit * 2.5;
+    }
+
+    &:not(:first-child) {
+        margin-top: 20px;
+    }
+
+    &__content {
+        position: relative;
+        padding-left: $txt-left-padding-desk;
+
+        &-heading {
+            margin-bottom: 12px;
+            @include bold-font(26);
+        }
+
+        &-description {
+            margin-top: 0;
+
+            @include core-font(18);
+
+            p:not(:first-child) {
+                margin-top: 20px;
+            }
+        }
+    }
+
+    &__icon-wrapper {
+        position: absolute;
+        top: 0;
+        left: 0;
+        padding-left: $box-padding;
+        width: 100%;
+        height: 100%;
+    }
+
+    &__icon {
+        position: absolute;
+        width: 100px;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+
+    @include media($max-width: $min-desktop-width, $ignore-for-ie: true) {
+        &__content {
+            padding-left: 0px;
+
+            p:not(:first-child) {
+                margin-top: 14px;
+            }
+
+            &-heading {
+                padding-left: $txt-left-padding-mob;
+            }
+        }
+
+        &__icon {
+            position: relative;
+            width: 36px;
+            top: 18px;
+            transform: none;
+        }
+    }
+
+    &--information,
+    &--important,
+    &--critical {
+        overflow-y: auto;
+
+        svg {
+            fill:none;
+            stroke-linecap:round;
+            stroke-linejoin:round;
+            stroke-width:4px;
+        }
+    }
+
+    &--information,
+    &--important {
+        svg {
+            stroke: $nhs-blue;
+        }
+    }
+
+    &--information {
+        border-left-color: $col-information-border;
+        background-color: $col-information-background;
+    }
+
+    &--important {
+        border-left-color: $col-important-border;
+        background-color: $col-important-background;
+    }
+
+    &--critical {
+        border-left-color: $col-critical-border;
+        background-color: $col-critical-background;
+        color: $white;
+
+        svg {
+            stroke: $white;
```

Comment:
The inline SVGs and there CSS are probably okay, but note we also have a server-side /svg-magic tool.  Take a look at the example below, it may come in handy.
```text
/svg-magic/${icon.path}?colour=005eb8
```

Reply:
That's good to know, thanks for the heads up @GeoffHayward